scan/cli.go

```diff
@@ -110,6 +125,40 @@ func GetCommands() []cli.Command {
 	})
 }
 
+func AuditCmd(c *cli.Context) error {
+	wd, err := os.Getwd()
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+	detectedTechnologies, err := coreutils.DetectTechnologies(wd, false, false)
+	if err != nil {
+		return err
+	}
+	for tech, detected := range detectedTechnologies {
+		if detected {
+			log.Info(string(tech) + " detected.")
+			switch tech {
+			case coreutils.Maven:
+				err = AuditMvnCmd(c)
+			case coreutils.Gradle:
+				err = AuditGradleCmd(c)
+			case coreutils.Npm:
+				err = AuditNpmCmd(c)
+			case coreutils.Go:
+				err = AuditGoCmd(c)
+			case coreutils.Pypi:
+				err = AuditPipCmd(c)
+			default:
+				log.Info("Unfortunately " + string(tech) + " is not supported at the moment.")
```

Comment:
log.Info("Unfortunately " + string(tech) + " is not supported at the moment.")
-->
log.Info(string(tech), " is currently not supported")